libs/wasm/detective/src/lib.rs

```diff
@@ -50,7 +50,7 @@ pub extern "C" fn f(ptr: *mut u8, length: usize) -> u64 {
                         special_fields: Default::default(),
                     },
                 )),
-                special_fields: Default::default(),
+                ..Default::default()
```

Comment:
ISR now requires audience, but that is filled out by the SDK, so just set all defaults here

Reply:
Let's add a ticket in backlog to remember to update all other SDKs to do it at some point.

Reply:
^ Can reference this issue.

Reply:
Added ENG-1747 for other sdks